lib/fs.js

```diff
@@ -319,39 +329,46 @@ fs.existsSync = function(path) {
   }
 };
 
+const defaultReadFileOpts = Object.create(null, {
+  flag: { value: O_RDONLY, writable: true }
+});
```

Comment:
That's expected because of the changes made in V8 to `Object.create(null)`, but at least it's only created once during startup, so the performance cost of creating the object is amortized.